src/app/public/styles/themes/modern/_buttons.scss

```diff
@@ -1,12 +1,43 @@
+@import '../../mixins';
 @import '../../variables';
 @import '../../variables-public';
 @import './_compat/mixins';
 
 .sky-theme-modern {
   .sky-btn {
+    border-radius: $sky-theme-modern-border-radius-md;
     font-size: $sky-theme-modern-font-body-default-size;
   }
 
+  .sky-btn-borderless,
+  .sky-btn-danger,
+  .sky-btn-default,
+  .sky-btn-link,
+  .sky-btn-primary {
+    padding: 8px 20px;
```

Comment:
Should we be using SASS variables for these values?

Reply:
I wasn't going to, but in styling toolbar for modern theme I noticed these values are also used there, so I created a design token for it.